components/zenFunctions/client/fn_zen_woundUnitRandomlyAdvanced.sqf

```diff
@@ -0,0 +1,27 @@
+#include "macros.hpp"
+
+params ["_unit"];
+
+if (isNull _unit) then
+{
+    systemChat "No unit found to wound.";
+}
+else
+{
+	["Wound Unit Randomly",
+	[
+		["SLIDER", "Max Number of Wounds", [1, 20, 6, 0]],
+		["COMBO", "Wound Types", [[["bullet", "grenade", "explosive", "shell", "vehiclecrash", "collision", "backblast", "stab", "punch", "falling", "ropeburn", "drowning"], ["bullet"], ["grenade"], ["explosive"], ["shell"], ["vehiclecrash"], ["collision"], ["backblast"], ["stab"], ["punch"], ["falling"], ["ropeburn"], ["drowning"]], ["All", "Bullet", "Grenade", "Explosive", "Shell", "Vehicle Crash", "Collision", "Backblast", "Stab", "Punch", "Fall", "Rope Burn", "Drowning"],0]],
```

Comment:
This dialogue box is super cool and I didn't know ZEN had this.

My concern is with the wound types spam and whether it could come from some central ACE variable or config instead so this doesn't fall out of date if they change anything.  However, I don't see the potential for that being high enough to justify a change until it does happen so this is fine.